README.md

```diff
@@ -1,70 +1,89 @@
-# Getting Started with Create React App
+<p align="center">
+  <a href="https://www.microverse.org/">
+    <img alt="Microverse" src="https://img.shields.io/badge/-Microverse-blueviolet?style=flat-square">
+  </a>
+  <a href="https://github.com/VanessaAoki/covid-tracker/blob/development/LICENSE">
+    <img alt="MIT Licensed" src="https://img.shields.io/github/license/VanessaAoki/covid-tracker?style=flat-square">
+  </a>
+  <a href="https://github.com/VanessaAoki/covid-tracker">
+    <img alt="GitHub last commit (branch)" src="https://img.shields.io/github/last-commit/VanessaAoki/covid-tracker/master?color=blue&style=flat-square">
+  </a>
+  <a href="https://github.com/VanessaAoki/covid-tracker">
+    <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/VanessaAoki/covid-tracker?color=pink&label=%E2%98%85%20stars%20&style=flat-square">
+  </a>
+  <a href="https://github.com/VanessaAoki">
+    <img alt="GitHub followers" src="https://img.shields.io/github/followers/VanessaAoki?color=yellow&logo=github&style=flat-square">
+  </a>
+</p>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Covid-tracker
+Covid-tracker is a website that allows users to have access to the latest covid data in Brazil. They can also filter by region name, as well as get more informations on that specific region by clicking on it.
+> Original design by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
```

Comment:
- [OPTIONAL] This would have been nice if it was moved into an Acknowledgement section of the README. What do you think?